setup.py

```diff
@@ -205,7 +205,7 @@ def do_setup():
             'flask-login==0.2.11',
             'flask-swagger==0.2.13',
             'flask-wtf==0.12',
-            'funcsigs>=1.0.2, <1.1',
```

Comment:
Mmm this ties it to the older version which would downgrade it at some other places. I'm not sure why we were dependent on >= 1.0.2, so I am not too happy by 'just' downgrading. We could also require a higher version of setup tools.